tests/customer-resource-custom-coverage-test.js

```diff
@@ -173,10 +175,10 @@ describeApplication('CustomerResourceCustomCoverage', () => {
 
       describe('entering an invalid date range', () => {
         beforeEach(() => {
-          return convergeOn(() => {
-            expect(CustomerResourceCoverage.dateRangeRowList[0].$beginCoverageField).to.exist;
-            expect(CustomerResourceCoverage.dateRangeRowList[0].$endCoverageField).to.exist;
-          });
+          return new Convergence()
+            .once(() => expect(CustomerResourceCoverage.dateRangeRowList[0].$beginCoverageField).to.exist)
+            .once(() => expect(CustomerResourceCoverage.dateRangeRowList[0].$endCoverageField).to.exist)
```

Comment:
Instead of `.once().once()` we can just lump these expectations into a single `.once()`